prow/create.sh

```diff
@@ -2,22 +2,25 @@
 
 set -ex
 
+NAMESPACE=${NAMESPACE:-default}
+WORKER_NS=${WORKER_NS:-test-pods}
+
 # create test-pods namespace
-kubectl create namespace test-pods || echo Skipping
+kubectl create "${NAMESPACE}" test-pods || echo Skipping
```

Comment:
this is wrong

Reply:
That's for the catch - that's what I meant in the comment above, but happy that someone has careful eyes :) Fixed in ef1fb7f910